app/views/admin/index.html.haml

```diff
@@ -20,6 +20,8 @@
       %a{href: '/sidekiq'} Job queue
     %li
       %a{href: 'https://dashboard-testing.wikiedu.org/rails/mailers'} Email templates
+    %li
+      %a{href: '/course_copy_prod'} Copy course from production
```

Comment:
Probably a better label for this will be "Copy course from another Dashboard server"

Reply:
Updated here [82d04a6](https://github.com/WikiEducationFoundation/WikiEduDashboard/pull/4914/commits/82d04a6c1dbb8c370b5f958e0134a994b43b98ef)